packages/suite/src/views/wallet/transactions/index.tsx

```diff
@@ -36,9 +37,18 @@ const Content = ({ selectedAccount, children }: ContentProps) => (
     </WalletLayout>
 );
 
+interface DataProviderContext {
+    supportedTokenIcons: string[] | null;
+}
+
+export const DataProviderContext = React.createContext<DataProviderContext>({
```

Comment:
i wouldn't use this provider as a "global data provider" while it's used only for icons (and probably will be removed soon)